packages/keychain/src/components/ErrorAlert.tsx

```diff
@@ -367,13 +350,8 @@ function StackTraceDisplay({
                     </Text>
                     {key === "address" || key === "class" ? (
                       <Link
-                        to={
-                          getExplorerUrl(
-                            key === "address" ? "contract" : "class",
-                            value as string,
-                          ) as string
-                        }
-                        target={isExternalLink ? "_blank" : undefined}
+                        to={explorer.transaction(value as string)}
```

Comment:
hmm i dont think this is a transaction right? previously it was either contract or class

Reply:
oops, my bad